examples/dotnet-module/README.md

```diff
@@ -0,0 +1,10 @@
+
+## Minimal Example .NET Node Module
+The `Example.cs` class defines a Node.js add-on module that runs on .NET. The `example.js` script loads that .NET module and calls a method on it. The script has access to type definitions and doc-comments for the module's APIs via the auto-generated `.d.ts` file.
```

Comment:
>The [](http://example.com/codeflow?start=0&length=3)

nit: can we avoid long text lines? MD format allows using new lines in the same paragraph.

Reply:
Fixed in all the MD files.